src/ui/components/results/verticalresultscomponent.js

```diff
@@ -92,7 +92,7 @@ class VerticalResultsConfig {
        * Any fieldIds in hiddenFields will be hidden from the list of appied filters.
        * @type {Array<string>}
        */
-      hiddenFields: defaultConfigOption(config, ['appliedFilters.hiddenFields', 'hiddenFields'], ['builtin.location']),
```

Comment:
This used to be part of another PR. Was there a rebasing issue?

Reply:
Ah I noticed github automatically rebases relation changes I wanted to try something like setting this PR as the base for that second PR. I guess that only works if this one gets merged in first, though. I'll rebase!